### PR TITLE
add cafe list

### PR DESCRIPTION
**Pull request by MaxAstin**

### CODE REVIEW

File: app/src/main/java/com/bunbeauty/fooddeliveryadmin/screen/cafelist/CafeListFragment.kt [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+
+    @Composable
+    private fun CafeListScreen(
+        uiState: CafeListUiState,
+        onRetryClicked: () -> Unit,
+        onCafeClicked: (CafeUiItem) -> Unit
+    ) {
+        AdminScaffold(
+            title = stringResource(R.string.title_cafe_list),
+            backActionClick = { findNavController().popBackStack() }
```
Comment (xidealo) [inline app/src/main/java/com/bunbeauty/fooddeliveryadmin/screen/cafelist/CafeListFragment.kt:65]: back action тоже можно передать

File: app/src/main/java/com/bunbeauty/fooddeliveryadmin/screen/cafelist/CafeListFragment.kt [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+            }
+        }
+    }
+
+    @Composable
+    private fun CafeListSuccessScreen(
+        cafeList: List<CafeWithWorkingHours>,
+        onCafeClicked: (CafeUiItem) -> Unit
+    ) {
+        val cafeItems = remember(cafeList) {
```
Comment (xidealo) [inline app/src/main/java/com/bunbeauty/fooddeliveryadmin/screen/cafelist/CafeListFragment.kt:90]: something new?

File: data/src/main/java/com/bunbeauty/data/repository/CafeRepository.kt [hunk 3]
```diff
@@ -4,37 +4,31 @@ import com.bunbeauty.common.extension.getNullableResult
...
 
     override suspend fun getCafeListByCityUuid(cityUuid: String): List<Cafe> = withContext(IO) {
-        cafeDao.getCafeListByCityUuid(cityUuid).map { cafeEntity ->
-            entityCafeMapper.from(cafeEntity)
-        }
+        cafeDao.getCafeListByCityUuid(cityUuid).map(cafeMapper::map)
     }
 
-    override suspend fun getCafeList(token: String, cityUuid: String): List<Cafe> {
+    override suspend fun getCafeList(cityUuid: String): List<Cafe> {
```
Comment (xidealo) [inline data/src/main/java/com/bunbeauty/data/repository/CafeRepository.kt:28]: =\ what if data was expired?

File: presentation/src/main/java/com/bunbeauty/presentation/feature/settings/SettingsViewModel.kt [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+    private val getIsUnlimitedNotification: GetIsUnlimitedNotificationUseCase,
+    private val updateIsUnlimitedNotification: UpdateIsUnlimitedNotificationUseCase,
+): BaseViewModel() {
+
+    private val mutableIsUnlimitedNotifications = MutableStateFlow<Boolean?>(null)
+    val isUnlimitedNotifications = mutableIsUnlimitedNotifications.asStateFlow()
+
+    fun updateData() {
+        viewModelScope.launchSafe(
+            onError = {},
```
Comment (xidealo) [inline presentation/src/main/java/com/bunbeauty/presentation/feature/settings/SettingsViewModel.kt:25]: empty block?

**Comment by xidealo** [comment]
add testing pipeline to CI pls

File: presentation/src/main/java/com/bunbeauty/presentation/feature/settings/SettingsViewModel.kt (hunk 4 above)
Comment (MaxAstin) [reply presentation/src/main/java/com/bunbeauty/presentation/feature/settings/SettingsViewModel.kt:25]: Yes

File: app/src/main/java/com/bunbeauty/fooddeliveryadmin/screen/cafelist/CafeListFragment.kt (hunk 1 above)
Comment (MaxAstin) [reply app/src/main/java/com/bunbeauty/fooddeliveryadmin/screen/cafelist/CafeListFragment.kt:65]: Fixed

**Pull request merged**